conu/apidefs/metadata.py

```diff
@@ -0,0 +1,48 @@
+class Metadata:
+    """
+    Common metadata for container and image
+    """
+
+    def __init__(self, name=None, id=None, ports=None, labels=None, command=None,
+                 creation_timestamp=None, env_variables=None):
+        self.name = name
+        self.id = id
```

Comment:
please use `identifier` since `id` is a builtin python function:
```
In [1]: help(id)
Help on built-in function id in module __builtin__:

id(...)
    id(object) -> integer
    
    Return the identity of an object.  This is guaranteed to be unique among
    simultaneously existing objects.  (Hint: it's the object's memory address.)
```